src/main.c

```diff
@@ -41,13 +41,13 @@
 #include "log.h"
 
 #define AUDIO_BUFFERS 128
-#define AUDIO_THRESHOLD 40
-#define AUDIO_DATA_LENGTH 8192
+#define MAX_AUDIO_DATA_LENGTH 31072
```

Comment:
Where does this number come from?

Reply:
Good question. This is the maximum size this PR outputs due to the SDR chuck size `(512 * 1204)` defined in nrsc5.c.
Basically: `(512 * 1024) / 1488375 * 44100 * sizeof(int)` rounded up a little bit.